homeassistant/components/pulseaudio_loopback/switch.py

```diff
@@ -1,52 +1,33 @@
 """Switch logic for loading/unloading pulseaudio loopback modules."""
-from datetime import timedelta
 import logging
-import re
-import socket
 
+from pulsectl import Pulse, PulseError
 import voluptuous as vol
 
-from homeassistant import util
 from homeassistant.components.switch import PLATFORM_SCHEMA, SwitchEntity
 from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
 import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 _PULSEAUDIO_SERVERS = {}
 
-CONF_BUFFER_SIZE = "buffer_size"
 CONF_SINK_NAME = "sink_name"
 CONF_SOURCE_NAME = "source_name"
-CONF_TCP_TIMEOUT = "tcp_timeout"
 
 DEFAULT_BUFFER_SIZE = 1024
 DEFAULT_HOST = "localhost"
 DEFAULT_NAME = "paloopback"
-DEFAULT_PORT = 4712
-DEFAULT_TCP_TIMEOUT = 3
+DEFAULT_PORT = 4713
 
 IGNORED_SWITCH_WARN = "Switch is already in the desired state. Ignoring."
 
-LOAD_CMD = "load-module module-loopback sink={0} source={1}"
-
-MIN_TIME_BETWEEN_FORCED_SCANS = timedelta(milliseconds=100)
-MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
-MOD_REGEX = (
-    r"index: ([0-9]+)\s+name: <module-loopback>"
-    r"\s+argument: (?=<.*sink={0}.*>)(?=<.*source={1}.*>)"
-)
-
-UNLOAD_CMD = "unload-module {0}"
-
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_SINK_NAME): cv.string,
         vol.Required(CONF_SOURCE_NAME): cv.string,
-        vol.Optional(CONF_BUFFER_SIZE, default=DEFAULT_BUFFER_SIZE): cv.positive_int,
         vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
```

Comment:
Why you need now that settings? He read the local settings from /etc/pulse and do the right thing. However, we can have such settings optional

Reply:
You're right. I updated the code to default to using the configuration in /etc/pulse....